pkg/devfile/parser/data/v2/volumes.go

```diff
@@ -8,99 +8,90 @@ import (
 	"github.com/devfile/library/pkg/devfile/parser/data/v2/common"
 )
 
-// AddVolume adds the volume to the devFile and mounts it to all the container components
-func (d *DevfileV2) AddVolume(volumeComponent v1.Component, path string) error {
-	volumeExists := false
+// AddVolumeMount adds the volume mount to the specified container component
+func (d *DevfileV2) AddVolumeMount(componentName, name, path string) error {
```

Comment:
since other APIs take go struct if it's ever defined in devfile api, suggest to use `v1.VolumeMount` to be consistent with other APIs